### ISSUE TITLE
artman smoke tests failing due to ascii encoding error

### ISSUE THREAD

**Issue by andreamlin** [googleapis/artman#537]
https://github.com/googleapis/artman/pull/536 may have broken the artman_smoke_tests CircleCI job. This is affecting gapic-generator ([failure](https://circleci.com/gh/googleapis/gapic-generator/12292#artifacts/containers/0)) and artman (https://circleci.com/gh/googleapis/artman/8656)

[Failure log](https://12292-56533838-gh.circle-artifacts.com/0/tmp/smoketest.log) has failure stack traces that point to [protoc_utils.py:212](https://github.com/googleapis/artman/pull/536/files#diff-9d8085d371144502c87df6a0b16b5031R212), which was modified in the latest artman PR:
```
  File "/tmp/artman/artman/cli/main.py", line 71, in main
    engine.run()
  File "/usr/local/lib/python3.5/dist-packages/taskflow/engines/action_engine/engine.py", line 159, in run
    for _state in self.run_iter():
  File "/usr/local/lib/python3.5/dist-packages/taskflow/engines/action_engine/engine.py", line 223, in run_iter
    failure.Failure.reraise_if_any(it)
  File "/usr/local/lib/python3.5/dist-packages/taskflow/types/failure.py", line 292, in reraise_if_any
    failures[0].reraise()
  File "/usr/local/lib/python3.5/dist-packages/taskflow/types/failure.py", line 299, in reraise
    six.reraise(*self._exc_info)
  File "/usr/local/lib/python3.5/dist-packages/six.py", line 693, in reraise
    raise value
  File "/usr/local/lib/python3.5/dist-packages/taskflow/engines/action_engine/executor.py", line 82, in _execute_task
    result = task.execute(**arguments)
  File "/tmp/artman/artman/tasks/protoc_tasks.py", line 127, in execute
    excluded_proto_path=excluded_proto_path)
  File "/tmp/artman/artman/tasks/protoc_tasks.py", line 96, in _execute_proto_codegen
    protoc_utils.find_protos(src_proto_path, excluded_proto_path)).items():
  File "/tmp/artman/artman/utils/protoc_utils.py", line 227, in group_by_go_package
    pkg = go_pkg(file)
  File "/tmp/artman/artman/utils/protoc_utils.py", line 212, in go_pkg
    for line in f:
  File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
```

I can't repro this locally, but I can repro it in a CircleCi instance.

**Comment by andreamlin** [comment]
@pongad Can you investigate?

**Comment by pongad** [comment]
I'll revert this for now.

**Comment by pongad** [comment]
Fixed by #539.
